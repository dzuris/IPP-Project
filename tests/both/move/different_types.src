.ippcode22

DEFVAR GF@temp

MOVE GF@temp int@5
WRITE GF@temp

MOVE GF@temp bool@true
WRITE GF@temp

MOVE GF@temp string@\032text\032with\032spaces\010
WRITE GF@temp