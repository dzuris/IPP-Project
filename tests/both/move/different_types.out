5true text with spaces
